src/devices/cpu/i386/i386segs.hxx

```diff
@@ -2507,7 +2507,7 @@ inline void i386_device::dri_changed()
 				}
 				if(true_mask == 0)
 				{
-					logerror("i386: Unknown breakpoint length value\n");
+					LOGMASKED(LOG_GENERAL, "i386: Unknown breakpoint length value\n");
```

Comment:
This should probably be a `logerror` since it’s an actual error condition, so you probably want to know about it.  It‘s also an infrequently-used CPU feature, so it shouldn’t cause spam.

Reply:
Swapped that out.